apps/projects/app/components/Panel/FundIssues/FundIssues.js

```diff
@@ -24,8 +25,7 @@ import {
   Info,
 } from '@aragon/ui'
 
-import { Form, FormField, FieldTitle } from '../../Form'
-import { DateInput } from '../../../../../../shared/ui'
+import { Form, FormField, FieldTitle, DateInput } from '../../Form'
```

Comment:
It is interesting to keep using shared components instead of local ones, to avoid duplication, for example, this work will be repeated once we want to also update the `DateInput` component in `Rewards` or other apps

Reply:
Yes. Created issue #1406 for it.